index.html

```diff
@@ -1,183 +1,284 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="https://kit.fontawesome.com/e520ead38a.js" crossorigin="anonymous"></script>
-</head>
-<body>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/e520ead38a.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
     <header class="header">
-        <nav class="navbar">
-            <div class="web-name">Sohail</div>
-                <a href="#" class="bar-button">
-                <i class="fa-solid fa-bars"></i>
-            </a>
-        </nav>
+      <nav class="navbar">
+        <div class="web-name">Sohail</div>
+        <ul class="menu">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <a href="#" class="bar-button">
+          <i class="fa-solid fa-bars"></i>
+        </a>
+      </nav>
```

Comment:
[OPTIONAl] I suggest you fix your mobile design to match figma template. The desktop version should not affect the mobile design but should complement each other. Currently, the mobile design, the background images are overriding the test in mobile version
<img width="1676" alt="Screenshot 2022-12-08 at 15 34 39" src="https://user-images.githubusercontent.com/48546992/206448251-e1eee6e3-a710-4157-a361-de5cb48c248d.png">
Please fix this.